session/session_test.go

```diff
@@ -3219,3 +3225,34 @@ func (s *testSessionSuite2) TestPerStmtTaskID(c *C) {
 
 	c.Assert(taskID1 != taskID2, IsTrue)
 }
+
+func (s *testBackupRestoreSuite) TestBackupAndRestore(c *C) {
+	// only run BR SQL integration test with tikv store.
+	if *withTiKV {
+		cfg := config.GetGlobalConfig()
+		cfg.Store = "tikv"
+		cfg.Path = s.pdAddr
+		config.StoreGlobalConfig(cfg)
+		tk := testkit.NewTestKitWithInit(c, s.store)
+		tk.MustExec("create database if not exists br")
+		tk.MustExec("use br")
+		tk.MustExec("create table t1(v int)")
+		tk.MustExec("insert into t1 values (1)")
+		tk.MustExec("insert into t1 values (2)")
+		tk.MustExec("insert into t1 values (3)")
+
+		tk.MustQuery("select count(*) from t1").Check(testkit.Rows("3"))
+
+		os.RemoveAll("/tmp/bk1")
```

Comment:
How about using API like [os.TempDir](https://pkg.go.dev/os?tab=doc#TempDir) and [path.Join](https://pkg.go.dev/path?tab=doc#Join) to make the code more general?

Reply:
good idea